src/FSharp.Core/array.fsi

```diff
@@ -3252,6 +3252,301 @@ module Array =
+
+        /// <summary>Returns the greatest of all elements of the array, compared via Operators.max on the function result.</summary>
+        ///
+        /// <remarks>Throws ArgumentException for empty arrays.</remarks>
+        ///
+        /// <param name="projection">The function to transform the elements into a type supporting comparison.</param>
+        /// <param name="array">The input array.</param>
+        ///
+        /// <exception cref="T:System.ArgumentNullException">Thrown when the input array is null.</exception>
+        /// <exception cref="T:System.ArgumentException">Thrown when the input array is empty.</exception>
+        ///
+        /// <returns>The maximum element.</returns>
+        ///
+        /// <example id="para-maxby-1">
+        /// <code lang="fsharp">
+        /// let inputs = [| "aaa"; "b"; "cccc" |]
+        ///
+        /// inputs |> Array.Parallel.maxBy (fun s -> s.Length)
+        /// </code>
+        /// Evaluates to <c>"cccc"</c>
+        /// </example>
+        ///
+        /// <example id="para-maxby-2">
+        /// <code lang="fsharp">
+        /// let inputs: string[]= [| |]
+        ///
+        /// inputs |> Array.Parallel.maxBy (fun s -> s.Length)
+        /// </code>
+        /// Throws <c>System.ArgumentException</c>.
+        /// </example>
+        [<CompiledName("MaxBy")>]
+        [<Experimental("Experimental library feature, requires '--langversion:preview'")>]
+        val inline maxBy: projection: ('T -> 'U) -> array: 'T[] -> 'T when 'U: comparison
+
+        /// <summary>Returns the lowest of all elements of the array, compared via Operators.min.</summary>
+        ///
+        /// <remarks>Throws ArgumentException for empty arrays</remarks>
+        ///
+        /// <param name="array">The input array.</param>
+        ///
+        /// <exception cref="T:System.ArgumentNullException">Thrown when the input array is null.</exception>
+        /// <exception cref="T:System.ArgumentException">Thrown when the input array is empty.</exception>
+        ///
+        /// <returns>The minimum element.</returns>
+        ///
+        /// <example id="para-min-1">
+        /// <code lang="fsharp">
+        /// let inputs = [| 10; 12; 11 |]
+        ///
+        /// inputs |> Array.Parallel.min
+        /// </code>
+        /// Evaluates to <c>10</c>
+        /// </example>
+        ///
+        /// <example id="min-2">
+        /// <code lang="fsharp">
+        /// let inputs: int[]= [| |]
+        ///
+        /// inputs |> Array.Parallel.min
+        /// </code>
+        /// Throws <c>System.ArgumentException</c>.
+        /// </example>
+        [<CompiledName("Min")>]
+        [<Experimental("Experimental library feature, requires '--langversion:preview'")>]
+        val inline min: array: 'T[] -> 'T when 'T: comparison
+
+        /// <summary>Returns the lowest of all elements of the array, compared via Operators.min on the function result.</summary>
```

Comment:
Same here - should be different from the Min summary

Reply:
Why and how?

Reply:
Yes. min vs minBy should be different.